cryptus-app/components/graph/graph.module.scss

```diff
@@ -1,8 +1,18 @@
-.chart_body {
-    background: #00151c;
+.container {
+  flex: 1 1 auto;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  height: 100%;
+}
+
+.chart_line {
+  flex: 0 0 auto;
 }
 
-.chart {
-    background: #0e3040;
-    color: #67809f;
+.chart_bar {
+  flex: 0 0 0 auto;
+  max-height: 32px;
```

Comment:
why max-height and not height? Its not good to use max or min instead of the real (ex: height) because the ui could change when we don't want it to.

Reply:
le graph dépasse toujours sauf si je mets un max height